dali/base/dadfs.hpp

```diff
@@ -432,11 +432,14 @@ interface IDistributedSuperFile: extends IDistributedFile
     virtual IDistributedFile *querySubFileNamed(const char *name,bool sub=true)=0;
     virtual unsigned numSubFiles(bool sub=false)=0;
 
-    // useful part funtions
+    // useful part functions
     virtual bool isInterleaved()=0;
     virtual IDistributedFile *querySubPart(unsigned partidx,                // superfile part index
                                            unsigned &subfilepartidx)=0;     // part index in subfile
                                 // returns file for part (not linked) NULL if not found
+
+    virtual void reloadSubFiles()=0;
+    virtual void relinkSubFile(unsigned pos)=0;
```

Comment:
Not sure why you've extended the IDistributedSuperFile public interface..
Shouldn't need to, to construct a new CDistributedSuperFile object.